plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -57,7 +56,5 @@ open class KtlintExtension {
      *
      * Default is none, please, mirgrate to [reporters].
      */
-    @Deprecated(message = "Ktlint introduced multi output support since 0.11.1 version",
-            replaceWith = ReplaceWith("reporters = arrayOf()", "reporter"))
     var reporter: ReporterType? = null
```

Comment:
If you're removing the deprecated, shouldn't this be removed as well?

Reply:
@JLLeitschuh removed and force rebased